src/types.ts

```diff
@@ -20,6 +20,7 @@ export interface Field {
   type: ValueType;
   staticValue?: Value;
   documentation: string;
+  defaultValue: string;
```

Comment:
I guess it could be optional?

Reply:
I use string to avoid `@default `

Reply:
I use string to avoid case like empty `@default `

Reply:
I think we should throw error in this case? Or what do you think `@default ` means?

Reply:
Throw is good to me